### DISCUSSION TITLE
Add ability to add/remove x frames from beginning/end/both of selected/all segments

### DISCUSSION THREAD

**Discussion by visualpizza** [mifi/lossless-cut#1567] [Ideas]
Hi! I love this project and I mainly want to use it to cut out all silence as a first step when receiving raw footage, sometimes however I wish for an easy way to "feather" the selections the detect silence tool makes, I propose a function that allows you to add/remove these "safety margins" for all segments :)

This would help me to fine-tune the segments to my exact needs, so in the end I have a single file of merged segments of things emitting sound, but maybe with 2 seconds padding around it to have the ability to choose to show more context before a sound occurs, if that makes sense lol

**Comment by mifi** [comment]
that's a good idea! will conver tthis to a feature request
